chef/cookbooks/provisioner/templates/suse/crowbar_register.erb

```diff
@@ -55,6 +60,7 @@ while test $# -gt 0; do
         <% end -%>
         --gpg-auto-import-keys) CROWBAR_AUTO_IMPORT_KEYS=1 ;;
         --no-gpg-checks) CROWBAR_NO_GPG_CHECKS=1 ;;
+        --no-dhcp) USE_DHCP=0 ;;
```

Comment:
I probably wouldn't add the option (not really clear when this would be useful).

Reply:
Indeed, it was added to match the other options only. Removed it now.